api/products.thrift

```diff
@@ -0,0 +1,81 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+include "shared.thrift"
+
+namespace py ProductManagement
+namespace js ProductManagement
+
+const string API_VERSION = '6.0'
+
+
+/*
+struct PrivilegeRecord {
```

Comment:
This is in line with the not yet fully committed database tables for product access, in the `config_db_model`. I'd rather not remove this, as these structs and object models set forth the further development line.